Pipfile

```diff
@@ -15,4 +15,4 @@ vss-tools = {editable = true, path = "."}
 [dev-packages]
 
 [requires]
-python_version = "3.8.5"
+python_version = "3.8.12"
```

Comment:
In the README.md file we still mention 3.7 and 3.8.5 in some places, should they be replaced with 3.8.12 as well?

Reply:
Thanks @erikbosch. Fixed it with the latest update.